addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -200,6 +201,11 @@ var DebugManager = Widget.extend({
     split_assets: function() {
         window.location = $.param.querystring(window.location.href, 'debug=assets');
     },
+    clean_assets: function() {
+        var self = this;
+        ajax.jsonRpc('/web/clean_assets', 'call', {})
```

Comment:
I think it would be better to call directly the unlink method on ir.attachment from here, without adding a controller.

Reply:
Unlink needs a list of ids, so it will add an rpc call to search

Reply:
good point.  But this is not really a problem I think.  This is a rare operation, it can be done with 2 rpcs.  Adding a controller is a slightly more costly solution, because it increases the size of our public API.